app/views/holds/_holds_ready_table.html.erb

```diff
@@ -15,12 +15,12 @@
     <tbody>
     <% @holds_ready.each do |hold| %>
       <tr>
-        <td>
+        <td scope="row">
```

Comment:
```suggestion
        <td>
```
[`scope` only applies to `th`](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced)